main.go

```diff
@@ -222,17 +265,8 @@ func main() {
 	}
 
 	// print version on start
-	log.Printf("starting smtprelay, version: %s\n", VERSION)
-
-	if *logFile != "" {
```

Comment:
are we getting rid of logFile, if yes we should remove `logFile` from config.go and `log_file` config example .ini file.

Reply:
Hhmm yeah good point, I will address this

Reply:
Fixed, defaulting to `/dev/stdout` and writing to given file